src/main/java/teammates/ui/template/InstructorStudentListFilterBox.java

```diff
@@ -0,0 +1,24 @@
+package teammates.ui.template;
+
+import java.util.List;
+
+public class InstructorStudentListFilterBox {
+
+    private List<InstructorStudentListFilterCourse> courses;
+    private boolean displayArchive;
```

Comment:
I don't think so. But this is how I've been doing the migration all these while.